src/sql/magic_plot.py

```diff
@@ -77,7 +77,7 @@ def execute(self, line="", cell="", local_ns=None):
         """
         Plot magic
         """
-
+        line = util.change_to_double_quotations_in_windows(line)
```

Comment:
can you try with the solution I suggested in the original issue?

https://github.com/ploomber/jupysql/issues/425#issue-1677707236

```
import shlex

def magic_args(magic_execute, line):
    line = without_sql_comment(parser=magic_execute.parser, line=line)
    return magic_execute.parser.parse_args(shlex.split(line, posix=False))
```

your approach is valid but it might cause issues down the road, it's better to use shlex.split, if using that solution passes all tests, we can use it. if it doesn't, we can go with your solution

Reply:
Sorry, I didn't realize there is your suggestion. I tried it, and it worked.

Change in `parse.py` : [fixed](https://github.com/ploomber/jupysql/pull/822/files#diff-22cfc6ec3e879941494b36e0dffaf24c100ab50173c413d821ebab94d0627ddfR222)

This works from `%sql/%%sql` but not for `%sqlplot` and `%sqlcmd` so I had to made additional changes using `shlex`.
Change for `%sqlplot`: [fixed](https://github.com/ploomber/jupysql/pull/822/files#diff-769097f5e36be57ddec10c78df0900db8c1e254f5485bc94367aa7fcb07991c5R14)
Change for `%sqlcmd`: [fixed](https://github.com/ploomber/jupysql/pull/822/files#diff-2aa72d6ecafc2697ec8174353868dae816494131c70b7e48be6115fda9f61370R74)